static/css/global.css

```diff
@@ -211,12 +211,17 @@ a:focus {
   display: flex;
   justify-content: space-between;
   padding: 2em;
+  align-items: center;
```

Comment:
What is this actually doing? This class is flexed and you don't use text-align to position items in a flex container. You use `align-items` or `justify-content` the latter of which is already being used to push items to the outer edges.

Reply:
This is added here to align the items in center position. 

The position of the left arrow was not properly aligned.
![Screen Shot 2022-07-06 at 12 52 03 pm](https://user-images.githubusercontent.com/13076255/177457923-39113d29-db7b-4887-b703-71f5726b5900.png)

________________________________________________________________________

After adding this:
![Screen Shot 2022-07-06 at 12 52 11 pm](https://user-images.githubusercontent.com/13076255/177457966-4bc0a7fd-31a8-49d4-a614-75b692ac6360.png)

Reply:
Sorry my bad. For some reason my brain read it as text-align